hybrid/README.md

```diff
@@ -0,0 +1,78 @@
+# Hybrid Node Attestor
```

Comment:
Add the build badge here, as in the `tpm_psat` README.

Reply:
Done